### PR TITLE
Added description for programming languages:

### PR DESCRIPTION
**Pull request by the-shivam-gupta**
The tooltip description for the following languages
    1.Cpp
    2.Kotlin
    3.Rust
    4.Dart
    5.Java
has been added.

## Fixes Issue
#1617

## Screenshots
image

### CODE REVIEW

**Review by k-deepak04 — DISMISSED** [review]

**Review by rupali-codes — DISMISSED** [review]
looks good to me

File: components/TopBar/CategoryDescriptions.ts [hunk 1]
```diff
@@ -35,6 +35,11 @@ const categoryDescriptions: CategoryDescriptions = {
   validation: 'Validation is the process of checking whether a system or its component(s) satisfies the specified requirements or not and checks the data is correct or not.',
   // languages
   'c programming': 'C is a general-purpose programming language that was developed in the early 1970s by Dennis Ritchie at Bell Labs. C is widely used for developing system software, embedded systems, and application software.',
+  cpp: "C++ is a versatile and powerful programming language that builds upon the features of C, offering object-oriented programming capabilities and support for low-level memory manipulation. It's widely used for developing system software, games, high-performance applications, and more.",
+  dart: "Dart is a programming language primarily used for building web and mobile applications, with a focus on client-side development. It's the language behind the Flutter framework, allowing developers to create cross-platform apps with a single codebase. Dart offers features like a Just-In-Time (JIT) compiler for development speed and an Ahead-Of-Time (AOT) compiler for production performance optimization.",
```
Comment (CBID2) [inline components/TopBar/CategoryDescriptions.ts:39]: [Typo]
```suggestion
  dart: "Dart is a programming language primarily used for building web and mobile applications with a focus on client-side development. It's the language behind the Flutter framework, allowing developers to create cross-platform apps with a single codebase. Dart offers features like a Just-In-Time (JIT) compiler for development speed and an Ahead-Of-Time (AOT) compiler for production performance optimization.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]

**Review by CBID2 — APPROVED** [review]
LGTM

**Comment by CBID2** [comment]
> @CBID2 
> 
> Please review this. 

Going to need your approval @Anmol-Baranwal

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Pull request merged**